pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -218,29 +204,67 @@ func (r *ReconcileChaosEngine) Reconcile(request reconcile.Request) (reconcile.R
 		reconcileEngine: engineReconcile,
 	}
 
-	monitorService := &serviceEngineMonitor{
-		service:         &corev1.Service{},
-		engineMonitor:   engineMonitor,
-		reconcileEngine: engineReconcile,
-	}
-
 	// Check if the engineRunner pod already exists, else create
 	err = engineRunnerPod(runnerPod)
 	if err != nil {
 		return reconcile.Result{}, err
 	}
```

Comment:
Make a line gap here, and put a comment for below function

Reply:
Cant add one more, linting isn't allowing